editor/test/integration/atlas_test.clj

```diff
@@ -47,6 +47,18 @@
               "pow_anim"]
              animation-ids-in-ddf)))))
 
+(deftest rename-anim
+  (test-util/with-loaded-project
+    (let [workspace (test-util/setup-scratch-workspace! world "test/resources/image_project")
+          project (test-util/setup-project! workspace)
+          atlas (project/get-resource-node project "/main/rename.atlas")
+          ddf-texture-set (g/node-value atlas :texture-set)
+          animation-ids-in-ddf (mapv :id (:animations ddf-texture-set))]
+      (is (= (set ["ball"
+                   "diamond_dogs"
+                   "test_anim"])
+             (set animation-ids-in-ddf))))))
+
```

Comment:
```suggestion
(deftest rename-anim
  (test-util/with-loaded-project "test/resources/image_project"
    (let [atlas (project/get-resource-node project "/main/rename.atlas")
          ddf-texture-set (g/node-value atlas :texture-set)
          animation-ids-in-ddf (into #{}
                                     (map :id)
                                     (:animations ddf-texture-set))]
      (is (= #{"ball"
               "diamond_dogs"
               "test_anim"}
             animation-ids-in-ddf)))))

```
* `test-util/with-loaded-project` will set up a `workspace` and a `project` for you. By default it will load the shared test project, but you can specify the desired project path. Scratch workspaces are used when you need to modify the project disk state in your test, but we don't need it here.
* You can map the `animation-ids-in-ddf` directly into a `set` using `(into <target-coll> <transducer> <source-coll>)`. Here `(map :id)` is used without an input collection argument, which returns a [transducer](https://clojure.org/reference/transducers) (kind of a recipe of one or more transformations) instead of a collection.
* You can use the [set literal](https://clojure.org/reference/data_structures#Sets) syntax for the expected result.